pytero/app.py

```diff
@@ -632,17 +632,26 @@ async def create_node_allocation(
         ip: str,
         ports: list[str],
         alias: str = None
-    ) -> Allocation:
-        data = await self._http.post(
+    ) -> None:
+        """ Create a new allocated node.
+
+        node: :class:`int`
+            The ID of the allocation.
```

Comment:
This isn't the allocation ID, it's the node ID. The allocation ID wouldn't exist yet.